### DISCUSSION TITLE
PCB Design

### DISCUSSION THREAD

**Discussion by nistvan86** [picocomputer/community#16] [Q&A]
First, congratulations for this project, I really like it.
I'm considering building one myself.

I saw your latest circuit board build video, and tried to find the PCB design, but I can't find it or it hasn't been released to the public. There's a mention of the PCB in the assembly document, but no link to it. I tried to find it in the repo but doesn't seem to be here. You said in the video it's some kind of contributions from your patrons. I'm not (yet) a patron of yours so I can't see posts there.

Is there any plan to release the board design, or should I just resort to using a prototype or veroboard to build mine for now?

**Comment by rumbledethumps** [comment]
The KiCad sources will be released after I validate some footprint changes. I'll announce its availability on [Twitter](https://twitter.com/Rumbledethumps) too.

**Reply by nistvan86** [reply]
Thank you! Then I'll wait a little more. :)

**Reply by rumbledethumps** [reply]
It's here. https://picocomputer.github.io/hardware.html

**Reply by nistvan86** [reply]
Amazing, thank you! :) Can't wait to have some time to take a closer look.

**Comment by Nordlaw-tech** [comment]
Ordered all my components today. Just waiting for the final PCB files now :)
